g2core/cycle_homing.cpp

```diff
@@ -458,17 +464,38 @@ static int8_t _get_next_axis(int8_t axis) {
         if (hm.axis_flags[AXIS_A]) {
             return (AXIS_A);
         }
+        if (hm.axis_flags[AXIS_B]) {
+            return (AXIS_B);
+        }
+        if (hm.axis_flags[AXIS_C]) {
+            return (AXIS_C);
+        }
     } else if (axis == AXIS_X) {
         if (hm.axis_flags[AXIS_Y]) {
             return (AXIS_Y);
         }
         if (hm.axis_flags[AXIS_A]) {
             return (AXIS_A);
         }
+        if (hm.axis_flags[AXIS_B]) {
+            return (AXIS_B);
+        }
+        if (hm.axis_flags[AXIS_C]) {
+            return (AXIS_C);
+        }
     } else if (axis == AXIS_Y) {
         if (hm.axis_flags[AXIS_A]) {
             return (AXIS_A);
         }
+        if (hm.axis_flags[AXIS_A]) {
```

Comment:
Oops.  Cut-n-paste error, duplicating the `AXIS_A` code.  I'll push a follow up commit directly to edge.